corehq/apps/app_manager/id_strings.py

```diff
@@ -90,6 +90,16 @@ def detail_column_header_locale(module, detail_type, column):
     )
 
 
+@pattern('m%d.%s.%s_%s_%d.header')
+def invisible_column_header_locale(module, detail_type, column):
+    return u"m{module.id}.{detail_type}.{d.model}_{field}_{d_id}.header".format(
```

Comment:
Its just a string mapping so used the existing representation as convention. What would be the purpose of uniqueness?
Are we considering multiple entries? A sort only field would come only once in app_strings.